src/test/java/com/opencart/tests/editAccount/EditAccountTest.java

```diff
@@ -53,8 +56,9 @@ public void changeUserEmailToNewToOld() {
     public void changeUserEmailToNewLogoutLoginChangeUserEmailToOld() {
         accountDashboardBL
                 .clickOnEditAccountButton()
-                .setNewEmail()
+                .setNewEmail(newEmail)
                 .verifyAccountEdit();
+        loginModel = LoginModelRepository.getLoginWithNewEmail(newEmail);
```

Comment:
I belive it is better to rewrite it in the next way: 
loginModel = LoginModelRepository.getValidLoginModel();
loginModel.set("newEmail")

As a result you do not need to create new methods(getLoginWithNewEmail, getLoginWithNewPassword) for it within LoginModelRepository